frontend/src/scenes/session-recordings/playlist/sessionRecordingsListPropertiesLogic.ts

```diff
@@ -15,41 +16,57 @@ export const sessionRecordingsListPropertiesLogic = kea<sessionRecordingsListPro
     })),
 
     actions({
-        loadPropertiesForSessions: (ids: string[]) => ({ ids }),
-        maybeLoadPropertiesForSessions: (ids: string[]) => ({ ids }),
+        loadPropertiesForSessions: (sessions: SessionRecordingType[]) => ({ sessions }),
+        maybeLoadPropertiesForSessions: (sessions: SessionRecordingType[]) => ({ sessions }),
     }),
 
     loaders(({ actions }) => ({
         recordingProperties: [
             [] as SessionRecordingPropertiesType[],
             {
-                loadPropertiesForSessions: async ({ ids }, breakpoint) => {
-                    const paramsDict = {
-                        session_ids: ids,
-                    }
-                    const params = toParams(paramsDict)
+                loadPropertiesForSessions: async ({ sessions }, breakpoint) => {
                     await breakpoint(100)
 
                     const startTime = performance.now()
-                    const response = await api.recordings.listProperties(params)
+                    const sessionIds = sessions.map((x) => `'${x.id}'`).join(',')
+
+                    const oldestTimestamp = sessions.map((x) => x.start_time).sort()[0]
+                    const newestTimestamp = sessions.map((x) => x.end_time).sort()[sessions.length - 1]
+
+                    const hogql: HogQLQuery = {
+                        kind: NodeKind.HogQLQuery,
+                        query: `SELECT properties.$session_id as session_id, any(properties) as properties
+                                FROM events
+                                WHERE event IN ['$pageview', '$autocapture']
+                                AND session_id IN [${sessionIds}]
+                                AND timestamp >= '${dayjs(oldestTimestamp).format('YYYY-MM-DD HH:mm:ss')}'
+                                AND timestamp <= '${dayjs(newestTimestamp).format('YYYY-MM-DD HH:mm:ss')}'
```

Comment:
Note that HogQL operates in the project timezone, does `dayjs(oldestTimestamp).format('YYYY-MM-DD HH:mm:ss')` print out the datetime in that timezone too?

Reply:
These are questions I have no answers for :D

Reply:
Little play locally shows that:
```
2023-06-28T13:19:23.329000Z  becomes 2023-06-28 15:19:23
```
which should mean this works as expected

Reply:
I think that might be using your device timezone (UTC > CEST), but I believe in this case it's not the _project_ timezone that's being used

Reply:
So I'm gonna conjecture (and very happy to be proved wrong) that your project timezone in the local environment is the default UTC, in which case the `timestamp <= {newestTimestamp}` condition is going to look _like_ it works properly (since the actual cutoff is two hours in the future). Or it's CEST, in which case the condition is going to look fully correct, but only because the project timezone is the same as the device one.

Reply:
Eugh. Any idea what the solution is here?

Reply:
The only reason to have the timestamp range is to stop us querying too much data unnecessarily. So, plus/minus 12 hours to either side so the timezone doesn't matter?

Reply:
Looking at [dayjs docs](https://day.js.org/docs/en/timezone/timezone), I think you just need to do this:
```typescript
const { timezone } = useValues(teamLogic)
[...]
dayjs(oldestTimestamp).tz(timezone).format('YYYY-MM-DD HH:mm:ss')`
```
Maybe would be useful to have some helper for this, since this seems to be a common case 🤔

Reply:
I've gone for the laziest option and made the boundaries wider 🤣 

I'll leave the good solution as an exercise for anyone foolish enough to do work on timezones 🥇

Reply:
That's okay, although I'm fairly solution what I proposed _is_ the good solution (`start_time`/`end_time` do include the `Z` suffix)

Reply:
Fun for ya https://github.com/PostHog/posthog/pull/16393